templates/article.html

```diff
@@ -0,0 +1,25 @@
+title: Article Title
+link: [https://crate.io/blog/crate-blog](https://crate.io/blog/crate-blog)
```

Comment:
link is generated automatically. I think it is some legacy stuff that is never used